src/displayapp/screens/Notifications.cpp

```diff
@@ -126,12 +129,31 @@ void Notifications::OnPreviewInteraction() {
   }
 }
 
+void Notifications::OnPreviewDismiss() {
+  notificationManager.Dismiss(currentId);
+  if (timeoutLine != nullptr) {
+    lv_obj_del(timeoutLine);
+    timeoutLine = nullptr;
+  }
+  currentItem.reset(nullptr);
+  dismissingNotification = true;
+  afterDismissNextMessageFromAbove = true; // show next message coming from below
```

Comment:
copy paste error, or do we need a next-message from above-below handling like in the "normal" notification view? What is the expected behavior after dismissing a preview-notification? I assumed to get back to the previous screen, not to the next notification.

What is the currently implemented behavior? Especially what happens when we have multiple messages and then dismissing the newest from the preview.

```suggestion
  afterDismissNextMessageFromAbove = true; // show next message coming from above
```

Reply:
The behaviour in the PR is to return the to watchface after dismissing the preview. This holds true no matter how many notifications are available.
And yes, copy-paste error regarding the comment... will fix...